pkg/store/copr/coprocessor.go

```diff
@@ -1113,20 +1145,62 @@ func (it *copIterator) Next(ctx context.Context) (kv.ResultSubset, error) {
 	return resp, nil
 }
 
+func (it *copIterator) liteSendReq() (resp *copResponse) {
+	defer func() {
+		r := recover()
+		if r != nil {
+			logutil.BgLogger().Error("copIteratorWork meet panic",
```

Comment:
The original `ctx` could be used for the logger..

Reply:
great, done.